base-offline/layouts/partials/video.html

```diff
@@ -0,0 +1,26 @@
+{{ $youtubeKey := trim .Params.video_metadata.youtube_id " "}}
```

Comment:
**Suggestion / Reminder:** `{{- -}}` to cut down on whitespace.

I don't really think it's worth changing or worrying about, since we don't have a way to enforce it and it is only occasionally annoying.